HLTriggerOffline/Egamma/python/HLTmultiTrackValidatorGsfTracks_cff.py

```diff
@@ -0,0 +1,54 @@
+import FWCore.ParameterSet.Config as cms
+
+from Validation.RecoTrack.HLTmultiTrackValidator_cfi import *
+hltGsfTrackValidator = hltMultiTrackValidator.clone(
+    label = [
+        "hltEgammaGsfTracks",
+        "hltEgammaGsfTracksUnseeded",
+    ],
+    label_tp_effic           = "trackingParticlesElectron",
+    label_tp_effic_refvector = cms.bool(True), 
+    dirName                  = cms.string('HLT/EG/Tracking/ValidationWRTtp/'),
+)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsEta.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsEta.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsPhi.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsPhi.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsPt.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsPt.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsVTXR.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsVTXR.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsVTXZ.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsVTXZ.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsEta.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsEta.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsPhi.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsPhi.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsPt.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsPt.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsVTXR.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsVTXR.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsVTXZ.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsVTXZ.minRapidity = cms.double(-3.0)
```

Comment:
Ok, fair point. Could you then add a comment explaining that?

Then the above could be also written in a less verbose way
```
hltGsfTrackValidator = hltMultiTrackValidator.clone(
    ...,
    histoProducerAlgoBlock = dict(
        TpSelectorForEfficiencyVsEta = dict(minRapidity=-3, maxRapidity=3),
        TpSelectorForEfficiencyVsPhi = dict(minRapidity=-3, maxRapidity=3),
        ...
    )
)
```
(I'd just ignore the `GpSelector*` as they're not used in practice)